python/cugraph/cugraph/link_prediction/jaccard.py

```diff
@@ -114,12 +123,15 @@ def jaccard(input_graph, vertex_pair=None, do_expensive_check=False, use_weight=
         adjacent vertices in the graph.
 
     do_expensive_check : bool, optional (default=False)
-        Deprecated, no longer needed
+        Deprecated.
+        Originally, when set to Ture, jaccard implementation checked if
+        the vertices in the graph are (re)numbered from 0 to V-1 where
+        V is the total number of vertices.
```

Comment:
Hmmm, this doesn't really tell them what the alternative is (if there is one) or why it's being deprecated.  How about this (for this docstring and all others)?
```suggestion
        Deprecated.
        This option added a check to ensure integer vertex IDs are sequential values from 0 to V-1. That check is now redundant because cugraph unconditionally renumbers and un-renumbers integer vertex IDs for optimal performance, therefore this option is deprecated and will be removed in a future version.
```